python/test/unit/test_domain_decompositions.py

```diff
@@ -7,7 +7,7 @@
 import arbor as arb
 
 # check Arbor's configuration of mpi and gpu
-gpu_enabled = arb.__config__["gpu"]
+gpu_enabled = arb.__config__["gpu"] != "none"
```

Comment:
Since this is a pretty common user operation, and string comparisons are very meh 🤷 , why not either:

* Use `None`, or `False` for gpu disabled
* Expose a `gpu_enabled` bool

Reply:
The only thing that could work would be `None`, although I am not a particular fan.
What I would have liked is an enum, but making one just for the config? I'll stick to the 
None + list of strings.

Reply:
If you're not a fan of any of that, leave it on `"none"` but add `gpu_enabled` so we can have `config["gpu_enabled"` checks, which is cleaner than strcomp.

Reply:
I am even less enthused by storing information in two fields. None or String it is.